pagopa/io-app
# fix: [#177905135] Fixed wrong font family

## Short description
Changed wrong font family with correct one

## How to test
Try to create a credit card and see if text input font family is "Titillium".
You can see also see the screenshots below, at the bottom there is the inspector that shows which font family is in use

**Before changes:**
<img width="300" alt="Screenshot 2021-04-28 at 09 44 16" src="https://user-images.githubusercontent.com/61834253/116367381-969c5f80-a807-11eb-95db-2e71e9604a9b.png">

**After changes:**
<img width="300" alt="Screenshot 2021-04-28 at 09 46 37" src="https://user-images.githubusercontent.com/61834253/116368112-4bcf1780-a808-11eb-9e85-ea2cacaa5fc1.png">

<img width="300" alt="Screenshot 2021-04-28 at 09 46 55" src="https://user-images.githubusercontent.com/61834253/116367918-1cb8a600-a808-11eb-8331-1c8d1f489e80.png">
